NuGet.Config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<configuration>
+    <packageSources>
+        <add key="BepInEx.Unity.IL2CPP" value="https://nuget.bepinex.dev/v3/index.json" />
+        <add key="AmongUs.GameLibs.Steam" value="https://nuget.bepinex.dev/v3/index.json" />
+        <add key="BepInEx.AutoPlugin" value="https://nuget.bepinex.dev/v3/index.json" />
+        <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+        <add key="Samboy" value="https://nuget.samboy.dev/v3/index.json" />
+    </packageSources>
+</configuration>
```

Comment:
This should be added to the `UpdateModForNewVerAU` branch

Reply:
hmmm